### ISSUE TITLE
Remove abstract empty string from call to pubmed.summary

### ISSUE THREAD

**Issue by stuartromanek** [CAYdenberg/node-ncbi#9]
Hey, thanks for your work on this module!

I've done custom implementations of using the NCBI API before and know there is a way to fetch a full publication as JSON by its ID.

As I understand, with your module I can grab pieces of a publication by ID but never the complete object, is that correct or am I missing a piece?

Thanks!

**Comment by CAYdenberg** [comment]
You can grab the full SUMMARY as JSON - and you can using this module as well. The full complement of data from NCBI is stored in under the `raw` key in the summary object. 

```js
pubmed.summary(20517925).then((paper) => {
  console.log(paper.raw);
});
```

The FULL TEXT of the article is not available as JSON - even directly from NCBI. You have to get it as XML. In node-ncbi v0.5 you can get it using the `fulltext` method, but as it's unparsed XML. I mostly added this to parse it with [Lens](https://github.com/elifesciences/lens).

Not sure if that answers your question ...

**Comment by stuartromanek** [comment]
right. If i call `summary` on an ID, the `abstract` value is always an empty string, so I can't get both the abstract and the full details in the same grab, can I? (specific to pulling from pubmed)

Unsure if the empty abstract value is something the api is doing or the module..

**Comment by CAYdenberg** [comment]
Hmm. Now that I look at this that is probably a bug. I shouldn't be putting an empty string in for the abstract because it isn't possible to get the abstract on the same call. Thanks for pointing that out.

So, yes, this is the intended behaviour, except that the abstract field shouldn't be in there at all.

**Comment by CAYdenberg** [comment]
Fixed in v0.5.1
